src/S3Backup.php

```diff
@@ -52,85 +48,7 @@ protected function putToStorage(string $name, $content): void
         $this->s3Client->putObject([
             'Bucket' => $this->bucket,
             'Key' => $this->path . $name,
-            'Body' => $content,
+            'Body' => !is_string($content) ? stream_get_contents($content) : $content,
```

Comment:
hm .... ak nie je string (je stream) -> tak ho napr preved na string a tak posli. 

nie je to chyba? teraz sa to vzdy prevedie na string  moze dojst pamat.

povodna verzia bola takto:
![image](https://user-images.githubusercontent.com/19371734/123477573-8681e000-d5fe-11eb-8e8e-8b22e12bfbae.png)

Reply:
@webrouse jj, možná tě napadne co s tím...
když dělám backup z aws projektu do S3 což je ten případ co máš screen tak se ukládá soubor na disk a pak se tady dává `fopen(...` což je stream

ale když teď nově můžu zálohovat na S3 projekt který je na azure tak `getContentStream()` taky generuje stejný stream ale při `putObjectu` to padně na chybě `Payload content length is required and can not be calculated.`

proto jsem to převedl na string

Reply:
ale našel jsem že místo `putObject` je lepší použít `upload`
https://github.com/aws/aws-sdk-js/issues/2961

který umí uploadovat všechny typy streamů